test/cpp/jit/test_subgraph_matcher.cpp

```diff
@@ -507,8 +509,57 @@ graph(%x):
   %z = my::op2(%x)
   return (%y, %z))IR",
       &pattern2);
+  // Not supported multi-output pattern
```

Comment:
why isn't this supported?

Reply:
Because not the whole pattern is covered by a traversal up from the first output (`%z = ...` is not visited). I will add a comment.